google/cloud/bigtable/testing/embedded_server_test_fixture.cc

```diff
@@ -36,8 +37,18 @@ void EmbeddedServerTestFixture::SetUp() {
   StartServer();
 
   grpc::ChannelArguments channel_arguments;
-  static std::string const prefix = "cbt-c++/" + version_string();
-  channel_arguments.SetUserAgentPrefix(prefix);
+  static std::string const user_agent_prefix = [] {
+    std::string agent = "cbt-c++/" + version_string();
+#if GOOGLE_CLOUD_CPP_HAVE_EXCEPTIONS
+    agent = " ex";
```

Comment:
+= ?

Reply:
Fixed, and then I noticed the file name, which made no sense so I changed the actual place where we set this (and refactored to a single location for good measure).